### PR TITLE
Prohibit database names longer than 63 characters.

### PR DESCRIPTION
**Pull request by vpetrovykh**
Fixes: #1158

### CODE REVIEW

File: edb/schema/database.py [hunk 1]
```diff
@@ -39,7 +42,24 @@ class DatabaseCommandContext(sd.ObjectCommandContext):
...
+    def _create_begin(
+        self,
+        schema: s_schema.Schema,
+        context: sd.CommandContext,
+    ) -> s_schema.Schema:
+        schema = super()._create_begin(schema, context)
+
+        # Validate that the database name is fewer than 64 characters
+        name = self.get_attribute_value('name')
+        if len(str(name)) > 63:
```
Comment (elprans) [inline edb/schema/database.py:55]: Please use a constant instead of a magic number.  We have `MAX_NAME_LENGTH` in `pointers.py`, factor it out somewhere.

Comment (1st1) [reply edb/schema/database.py:55]: `server/defines.py` would be an OK place since we have so many other constants there.

Comment (elprans) [reply edb/schema/database.py:55]: Yes, although I'd move it out or we'll be importing `server` everywhere.

Comment (vpetrovykh) [reply edb/schema/database.py]: I've factored out the 63 constant into `server/defines.py` from `database.py`, `pointers.py` and `common.py`.

File: tests/test_database.py [hunk 2]
```diff
@@ -45,3 +45,9 @@ class TestDatabase(tb.ConnectedTestCase):
             await conn.aclose()
         finally:
             await self.con.execute('DROP DATABASE mytestdb;')
+
+    async def test_database_create_02(self):
+        with self.assertRaisesRegex(edgedb.SchemaDefinitionError,
+                                    r'Database names longer than 63 '
```
Comment (elprans) [inline tests/test_database.py:51]: You missed a hardcoded spot

File: edb/schema/pointers.py [hunk 3]
```diff
@@ -27,6 +27,7 @@
 from edb.edgeql import ast as qlast
 from edb.edgeql import compiler as qlcompiler
 from edb.edgeql import qltypes
+from edb.server import defines
```
Comment (elprans) [inline edb/schema/pointers.py:30]: Let's move the defines out of `server`.  I don't like the cross-pollination

File: tests/test_database.py (hunk 2 above)
Comment (vpetrovykh) [reply tests/test_database.py:51]: OK, I've updated the test.

File: edb/schema/pointers.py (hunk 3 above)
Comment (vpetrovykh) [reply edb/schema/pointers.py]: OK, moved to a separate file in `schema`

**Pull request merged**